app/javascript/components/project_members/NewProjectMember.vue

```diff
@@ -53,16 +94,40 @@ export default {
   },
   data: function () {
     return {
+      search: "",
       selectedUser: null,
-      selectedRole: null
+      selectedRole: null,
+      roleDescriptions: [
+        'Edit, comment, and mark Controls as requiring review. Cannot sign-off or approve changes to a Control. Great for individual contributors.',
+        'Author and approve changes to a Control.',
+        'Full control of a Project. Lock Controls, revert controls, and manage project members.'
+      ]
     }
   },
   computed: {
-    formAction: function () {
-      return "/projects/" + this.project.id + "/project_members";
-    },
     authenticityToken: function() {
       return document.querySelector("meta[name='csrf-token']").getAttribute("content");
+    },
+    searchedAvailableMembers: function() {
+      let downcaseSearch = this.search.toLowerCase()
+      return this.available_members.filter(pm => pm.email.toLowerCase().includes(downcaseSearch));
```

Comment:
I think this should also cover the case where someone is trying to search by name